docs/graphql/manual/guides/integrations/auth0-jwt.rst

```diff
@@ -163,3 +163,42 @@ The configured unauthorized role will be used whenever an access token is not pr
 
 This can be useful for data that you would like anyone to be able to access and can be configured and restricted
 just like any other role.
+
+Sync Users from Auth0
+^^^^^^^^^^^^^^^^^^^^^
+
+Now, that you have created an Auth0 application and can signup/login, you will need a way to sync your users in Postgres as well.
+All you really need is the Auth0 ``user_id`` in something like a "users" table.
+
+Using Auth0 Rules again, add the following rule which will insert a new user everytime someone signs-up.
+
+.. note::
+
+   We need to use an ``upsert`` operation here because social logins do not distinguish between sign-up and login. Hence, we need to run this rule everytime a successful login is made and do nothing if the user already exists.
```

Comment:
```suggestion
   We need to use an ``upsert`` operation here because social logins do not distinguish between sign-up and login. Hence, we need to run this rule every time a successful login is made and do nothing if the user already exists.
```